MSVC: generated headers include the full types they hold in Reference<> members.

Reference<T> has a virtual destructor, and MSVC instantiates a class template's virtual functions
as soon as the class is instantiated, so a member like Optional<SortedVector<ManagedReference<
ActiveAreaPOD*>>> needs ActiveAreaPOD complete in every file that includes the header. GCC defers
that, so these imports were enough there. Switch the five that broke the MSVC build from import
(forward declaration) to include. None of the included headers pulls its includer back in.

diff --git a/MMOCoreORB/src/server/zone/managers/minigames/GamblingManager.idl b/MMOCoreORB/src/server/zone/managers/minigames/GamblingManager.idl
index 30360307cd..7d2f153acf 100644
--- a/MMOCoreORB/src/server/zone/managers/minigames/GamblingManager.idl
+++ b/MMOCoreORB/src/server/zone/managers/minigames/GamblingManager.idl
@@ -8,7 +8,7 @@ import engine.log.Logger;
 import server.zone.objects.creature.CreatureObject;
 import engine.util.Observer;
 import server.zone.objects.scene.SceneObject;
-import server.zone.objects.tangible.terminal.gambling.GamblingTerminal;
+include server.zone.objects.tangible.terminal.gambling.GamblingTerminal;
 import server.zone.managers.minigames.events.GamblingEvent;
 
 include system.util.VectorMap;
diff --git a/MMOCoreORB/src/server/zone/objects/region/space/SpaceSpawnArea.idl b/MMOCoreORB/src/server/zone/objects/region/space/SpaceSpawnArea.idl
index 6fa505344c..42565611a2 100644
--- a/MMOCoreORB/src/server/zone/objects/region/space/SpaceSpawnArea.idl
+++ b/MMOCoreORB/src/server/zone/objects/region/space/SpaceSpawnArea.idl
@@ -14,7 +14,7 @@ import server.zone.objects.scene.SceneObject;
 include server.zone.objects.ship.ShipObject;
 import system.lang.Time;
 import server.zone.objects.region.space.SpaceSpawnAreaObserver;
-import server.zone.managers.ship.SpaceSpawn;
+include server.zone.managers.ship.SpaceSpawn;
 import system.util.HashTable;
 
 include system.util.Vector;
diff --git a/MMOCoreORB/src/server/zone/objects/ship/PobShipObject.idl b/MMOCoreORB/src/server/zone/objects/ship/PobShipObject.idl
index bd29810006..9e42c9eb67 100644
--- a/MMOCoreORB/src/server/zone/objects/ship/PobShipObject.idl
+++ b/MMOCoreORB/src/server/zone/objects/ship/PobShipObject.idl
@@ -13,7 +13,7 @@ import engine.util.u3d.Vector3;
 import server.zone.Zone;
 import server.zone.objects.ship.ai.ShipAiAgent;
 import system.thread.ReadWriteLock;
-import server.zone.objects.tangible.ship.interiorComponents.ShipInteriorComponent;
+include server.zone.objects.tangible.ship.interiorComponents.ShipInteriorComponent;
 
 include server.zone.objects.ship.ShipObject;
 import server.zone.objects.ship.events.PobCellDotTask;
diff --git a/MMOCoreORB/src/server/zone/objects/ship/ShipObject.idl b/MMOCoreORB/src/server/zone/objects/ship/ShipObject.idl
index 8abb867361..54049199fd 100644
--- a/MMOCoreORB/src/server/zone/objects/ship/ShipObject.idl
+++ b/MMOCoreORB/src/server/zone/objects/ship/ShipObject.idl
@@ -7,7 +7,7 @@ package server.zone.objects.ship;
 
 import system.lang.Time;
 import server.zone.objects.tangible.TangibleObject;
-import server.zone.objects.ship.components.ShipComponent;
+include server.zone.objects.ship.components.ShipComponent;
 import server.zone.objects.ship.events.ShipRecoveryEvent;
 import server.zone.Zone;
 import engine.service.proto.BaseMessage;
diff --git a/MMOCoreORB/src/server/zone/objects/tangible/TangibleObject.idl b/MMOCoreORB/src/server/zone/objects/tangible/TangibleObject.idl
index 4d0427d429..958720d76c 100644
--- a/MMOCoreORB/src/server/zone/objects/tangible/TangibleObject.idl
+++ b/MMOCoreORB/src/server/zone/objects/tangible/TangibleObject.idl
@@ -10,7 +10,7 @@ import engine.log.Logger;
 include server.zone.packets.scene.AttributeListMessage;
 import server.zone.objects.creature.CreatureObject;
 import server.zone.objects.tangible.tool.repair.RepairTool;
-import server.zone.objects.area.ActiveArea;
+include server.zone.objects.area.ActiveArea;
 import server.zone.objects.tangible.threat.ThreatMap;
 import engine.util.u3d.Vector3;
 
